### PR TITLE
article list rendered

### PR DESCRIPTION
**Pull request by Kamz1991**

### CODE REVIEW

File: src/components/Articles.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    getArticles().then((articlesFromAPi) => {
+      console.log(articlesFromAPi, "<<<<<<<<");
+      setArticles(articlesFromAPi);
+    });
+  }, []);
+
+  return (
+    <main>
+      <h2>List of our artilces</h2>
+      {articles.map((article) => {
```
Comment (corkesw) [inline src/components/Articles.jsx:17]: You might want to render more of the information that is coming back from each article

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <p>
+              <Header />
```
Comment (corkesw) [inline src/App.js:16]: What is the purpose of this component? Is it supposed to be displayed as well as other components and so should be outside of a Route? At the moment it just contains a link to itself.

File: src/components/Articles.jsx (hunk 1 above)
Comment (Kamz1991) [reply src/components/Articles.jsx:17]: sure ill  look not it now ty

File: src/api.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const articlesApi = axios.create({
+  baseURL: "https://kamz-backend-projct.herokuapp.com/api",
+});
+
+export const getArticles = () => {
+  return articlesApi.get("/articles").then(({ data }) => {
+    console.log(data.articles, ">>>>>>>>");
```
Comment (corkesw) [inline src/api.js:9]: remove console.logs as you go

File: src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
```
Comment (corkesw) [inline src/components/Header.jsx:1]: I think this is the homepage? This is where you would want to present a list of all articles so consider which element the path of "/" is rendering.

File: src/App.js [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Nav />
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <p>
```
Comment (corkesw) [inline src/App.js:15]: These <p> tags around the components are unnecessary and should be removed

Comment (Kamz1991) [reply src/App.js:15]: hmmm true otherwise the homepage is nothing

**Pull request merged**